shuffle_sdk/shuffle_sdk.py

```diff
@@ -361,6 +363,21 @@ def __init__(self, redis=None, logger=None, console_logger=None):#, docker_clien
 
         # Make start time with milliseconds
         self.start_time = int(time.time_ns())
+        try:
+            if isinstance(self.action, str):
+                self.action = json.loads(self.action)
+
+            action_id = self.action.get("id")
+            fullKey = str(self.current_execution_id) + "-" + str(action_id)
+            if fullKey in pastAppExecutions:
+                self.logger.info(f"[INFO] Duplicate execution detected for execution id - {self.current_execution_id} and action - {self.action["id"]}")
```

Comment:
I suggest we make this an [ERROR] as it should NEVER EVER occur. That way we can see it easily.